helpfiles/all.txt

```diff
@@ -153,9 +153,10 @@ Other maintenance operations (usually not needed):
 
 Other operations:
 
-  help [ARGUMENT]	Access to the help system
-  version		Print the program version
-  remote-paths		Print a plain list of the remote package paths
+  help [ARGUMENT]	        Access to the help system
+  version		        Print the program version
+  remote-paths		        Print a plain list of the remote package paths
+  batch OPERATION ARGUMENT...   Execute an operation in batch mode.
```

Comment:
Put "ARGUMENT..." inside brackets to hint about arguments being optional depending on the operation.

Reply:
Addressed